3rdparty/libtim-vx/tim-vx.cmake

```diff
@@ -0,0 +1,73 @@
+set(TIMVX_COMMIT_HASH "bb164d7d95610b0ab40f120fb248443ae60d5f87")
+set(OCV_TIMVX_DIR "${OpenCV_BINARY_DIR}/3rdparty/libtim-vx")
+set(OCV_TIMVX_SOURCE_PATH "${OCV_TIMVX_DIR}/TIM-VX-${TIMVX_COMMIT_HASH}")
+
+# Download TIM-VX source code
+if(EXISTS "${OCV_TIMVX_SOURCE_PATH}")
+    message(STATUS "TIM-VX: Use cache of TIM-VX source code at ${OCV_TIMVX_SOURCE_PATH}")
+    set(TIMVX_FOUND ON)
+else()
+    set(OCV_TIMVX_FILENAME "${TIMVX_COMMIT_HASH}.zip")
+    set(OCV_TIMVX_URL "https://github.com/fengyuentau/TIM-VX/archive/")
```

Comment:
We are going to have a discussion with TIM-VX devs to work out a branch in TIM-VX repo for us to use.